.github/workflows/terraform-lint.yml

```
@@ -20,8 +21,18 @@ jobs:
       matrix:
         env:
           - env/production
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     steps:
+      - name: Install Terraform
+        run: >
+          wget -O - https://apt.releases.hashicorp.com/gpg
+          | sudo gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg
+
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+          | sudo tee /etc/apt/sources.list.d/hashicorp.list
+
+          sudo apt update && sudo apt install terraform
+
```

The other option is to allow and use [hashicorp/setup-terraform](https://github.com/hashicorp/setup-terraform)

```suggestion
      - uses: hashicorp/setup-terraform
```

This is my preference since it's simpler and we will have to install Terraform for the nextstrain.org [terraform-lint workflow](https://github.com/nextstrain/nextstrain.org/blob/master/.github/workflows/terraform-lint.yml) as well.

---

Since these are ephemeral machines, setting up the whole package management system (APT, dpkg) for a one-off seems way more than necessary. Terraform's also available as a single executable, e.g. https://releases.hashicorp.com/terraform/1.10.2/terraform_1.10.2_linux_amd64.zip, and we could simply download that and invoke it.

I think this would be nicer than the `hashicorp/setup-terraform` action if only because adding that action is effectively adding another dep.

---

> we could simply download that and invoke it.

Updated in https://github.com/nextstrain/infra/pull/35/commits/b83227e96104e2e2fe3bd0790ec72dc9a08e37b3